includes/amp-helper-functions.php

```diff
@@ -524,6 +524,8 @@ function amp_get_post_image_metadata( $post = null ) {
 
 	if ( has_post_thumbnail( $post->ID ) ) {
 		$post_image_id = get_post_thumbnail_id( $post->ID );
+	} elseif ( 'attachment' === $post->post_type ) {
```

Comment:
What if the attachment isn't an image? I think this should also include `wp_attachment_is( 'image', $post )`

Reply:
Thanks, commit [15ea3d](https://github.com/Automattic/amp-wp/pull/1176/commits/15ea3d94c8336c1d614384c6d817ea3dec8a17a4) ensures the attachment is an image.